src/tools/quemap/bspfile.c

```diff
@@ -577,7 +577,7 @@ void ParseEntities(void) {
 void UnparseEntities(void) {
 	char *buf, *end;
 	epair_t *ep;
-	char line[2048];
+	char line[2096];
 	int32_t i;
```

Comment:
Not really based on something but if look at this line

```
sprintf(line, "\"%s\" \"%s\"\n", key, value);
```
key and value are allowed to hold 2048 bytes in total and we still have " chars in between.
In real life this is not a problem probably because no key/value is 1024 byte long but it's a possible overflow